web-console/src/components/refresh-button/refresh-button.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { TimedButton } from '../timed-button/timed-button';
+
+export interface RefreshButtonProps extends React.Props<any> {
+  refresh: () => void;
+  localStorageKey?: any;
```

Comment:
I know this is wrong I just am not sure how to interpret what the type of a localstoragekey is

<img width="861" alt="Screen Shot 2019-06-17 at 6 15 38 PM" src="https://user-images.githubusercontent.com/37322608/59646470-f6198680-912b-11e9-8d59-a93cbf1eee3f.png">

Reply:
I think it is just `LocalStorageKeys`